### PR TITLE
Added Support to influxdbv3

### PR DESCRIPTION
**Pull request by Tanmai2002**
## Description

Fix: #7972 

## Type of change
InfluxDB is  Upgraded to Influx DB v3 as Latest. The Current Implementation Should Support V2,V3. InfluxDb v1 is currently archived and legacy. Kindly confirm if we need to persist connection to the same

(Please delete options that are not relevant)

- [ ] 🐛 Bug fix (non-breaking change which fixes an issue)
- [ ] ⚡ New feature (non-breaking change which adds functionality)
- [X ] 📢 Breaking change (fix or feature that would cause existing functionality not to work as expected)
- [ ] 📄 This change requires a documentation update

## Verification Process
image
image

To ensure the changes are working as expected:

 - [ ]   Test Location: Specify the URL or path for testing.
 - [ ]   Verification Steps: Outline the steps or queries needed to validate the change. Include any data, configurations, or actions required to reproduce or see the new functionality.

## Additional Media:

- [ ] I have attached a brief loom video or screenshots showcasing the new functionality or change.

## Checklist:

- [ ] My code follows the style guidelines(PEP 8) of MindsDB.
- [ ] I have appropriately commented on my code, especially in complex areas.
- [ ] Necessary documentation updates are either made or tracked in issues.
- [ ] Relevant unit and integration tests are updated or added.

### CODE REVIEW

**Review by ZoranPandovski — CHANGES_REQUESTED** [review]
Can you please solve the conflicts?

File: mindsdb/integrations/handlers/influxdb_handler/influxdb_handler.py [hunk 1]
```diff
@@ -44,16 +47,25 @@ def connect(self):
 
         Raises Expection if ping check fails
         """
+        # if self.is_connected is True:
```
Comment (ZoranPandovski) [inline mindsdb/integrations/handlers/influxdb_handler/influxdb_handler.py:50]: Can we remove the comments?

File: mindsdb/integrations/handlers/influxdb_handler/README.md [hunk 2]
```diff
@@ -16,7 +16,15 @@ The required arguments to establish a connection are,
 * `influxdb_token`: Authentication token for the hosted influxdb cloud instance
 * `influxdb_db_name`: Database name of the influxdb cloud instance
 * `influxdb_table_name`: Table name of the influxdb cloud instance
+* `org` : Organisation of the influxdb cloud instance
 
+
+## Install Dependencies
+
+```
+pip install influxdb3-python
```
Comment (bala-ceg) [inline mindsdb/integrations/handlers/influxdb_handler/README.md:25]: please add this Python library to the requirements.txt

File: mindsdb/integrations/handlers/influxdb_handler/influxdb_handler.py (hunk 1 above)
Comment (Tanmai2002) [reply mindsdb/integrations/handlers/influxdb_handler/influxdb_handler.py]: Done

**Review by ZoranPandovski — APPROVED** [review]

**Pull request merged**